lib/LedgerSMB/Template/LaTeX.pm

```diff
@@ -57,13 +57,14 @@ use strict;
 use Template::Latex;
 use Template::Parser;
 use LedgerSMB::Template::TTI18N;
+use LedgerSMB::Magic qw( UNI_aring UNI_Aring );
 use Log::Log4perl;
 use TeX::Encode::charmap;
 use TeX::Encode;
 
 BEGIN {
-    delete $TeX::Encode::charmap::ACCENTED_CHARS{chr(0x00c5)};
-    delete $TeX::Encode::charmap::ACCENTED_CHARS{chr(0x00e5)};
+    delete $TeX::Encode::charmap::ACCENTED_CHARS{chr(UNI_Aring)};
+    delete $TeX::Encode::charmap::ACCENTED_CHARS{chr(UNI_aring)};
```

Comment:
@einhverfr noticed that the character was (incorrectly?) encoded by TeX::Encode. So he removed the element from the hash. I think it's better if there had been a comment as to what situation and TeX::Encode version this problem applied to.

my preference would be that the new code used:

```perl
    delete $TeX::Encode::charmap::ACCENTED_CHARS{"\N{LATIN CAPITAL LETTER A WITH RING ABOVE}"};
    delete $TeX::Encode::charmap::ACCENTED_CHARS{"\N{LATIN SMALL LETTER A WITH RING ABOVE}"};
```